proto/buf/alpha/registry/v1alpha1/image.proto

```diff
@@ -29,6 +30,17 @@ message GetImageRequest {
   string owner = 1;
   string repository = 2;
   string reference = 3;
+
+  // Exclude files from imported buf modules in this image.
+  bool exclude_imports = 4;
+
```

Comment:
I did not end up including a "exclude_source_info" bool, as users can use a field mask to omit them. As the field masks are include only in this proposal, its not very ergonomic though as you have to specify everything else. Open to suggestion around the ux.

Reply:
Got it. An `exclude_source_info` parameter sounds nice for the symmetry with `buf build`, but it can be added later (if ever).

As for the field mask problem, I wonder if we should expose a `include_mask` and an `exclude_mask` that could be combined for both whitelisting and blacklisting behavior?

Reply:
Given the field mask issues described in https://github.com/bufbuild/buf/pull/885#discussion_r793136489, I don't want to use those so resolving.